odoo/tools/xml_utils.py

```diff
@@ -14,3 +14,31 @@ def check_with_xsd(tree_or_str, xsd_path):
         #import UserError only here to avoid circular import statements with tools.func being imported in exceptions.py
         from odoo.exceptions import UserError
         raise UserError('\n'.join(str(e) for e in xml_errors.error_log))
+
+def xml_node_chain(first_parent_node, tags_list, str_value=None, return_node=None):
```

Comment:
The return_node parameter is useful in that it allows less verbose code. For example, I use it in enterprise/account_sepa_direct_debit/account_payment.py, line 76, where I create the node chain CdtrSchmeId/id/PrtvId/Othr/Id with the creditor identifier as its value, and directly get node Othr, allowing to add other elements to it, with another node chain, line 77.